bin/setup/index.js

```diff
@@ -326,6 +326,80 @@ const RULES_MAP = {
       }
     }
 
+    static async enableApiKey(request) {
+      try {
+        const { accessToken, org, site, apiKeyId } = await request.json();
+        
+        if (!accessToken || !org || !site || !apiKeyId) {
+          return RequestHelper.errorResponse('accessToken, org, site, and apiKeyId are required');
+        }
+
+        const siteConfigEndpoint = `https://admin.hlx.page/config/${org}/sites/${site}.json`;
+        
+        console.log(`Getting site config from ${siteConfigEndpoint}`);
+        
+        // First, get the current site configuration
+        const getResponse = await fetch(siteConfigEndpoint, {
+          method: 'GET',
+          headers: {
+            'x-auth-token': accessToken
+          }
+        });
+
+        if (!getResponse.ok) {
+          const errorText = await getResponse.text();
+          return RequestHelper.errorResponse(`Failed to get site config: ${getResponse.status} ${getResponse.statusText} - ${errorText}`, getResponse.status);
+        }
+
+        const siteConfig = await getResponse.json();
+        
+        console.log(`Enabling API key ${apiKeyId} for site ${org}/${site}`);
+        
+        // Manipulate the JSON to add the API key ID
+        if (!siteConfig.access) {
+          siteConfig.access = {};
+        }
+        if (!siteConfig.access.admin) {
+          siteConfig.access.admin = {};
+        }
+        if (!siteConfig.access.admin.apiKeyId) {
+          siteConfig.access.admin.apiKeyId = [];
+        }
+        
+        // Add the API key ID if it's not already in the array
+        if (!siteConfig.access.admin.apiKeyId.includes(apiKeyId)) {
+          siteConfig.access.admin.apiKeyId.push(apiKeyId);
+        }
+
+        // Make the POST request to update the site configuration
+        const postResponse = await fetch(siteConfigEndpoint, {
```

Comment:
can we make the post to the /apiKey.json endpoint specifically, which is safer and wouldn't affect other site configs in any way?

Reply:
We already do that to create the key, but then we need to enable it for operation, and this is done in the site config.